lib/widgets/profile/feedback.dart

```diff
@@ -0,0 +1,175 @@
+                ),
+              ],
+            ),
+            Padding(
+              padding: const EdgeInsets.all(5.0),
+              child: Text(
+                'Title',
+                style: MyFonts.w500.setColor(kWhite).size(15),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(5.0),
+              child: TextFormField(
+                style: MyFonts.w500.size(15).setColor(kWhite),
+                controller: title,
+                maxLength: 20,
+                maxLines: 1,
+                decoration: InputDecoration(
+                    border: OutlineInputBorder(
+                        borderRadius: BorderRadius.circular(8)),
+                    fillColor: kAppBarGrey,
+                    filled: true,
+                    hintStyle: MyFonts.w500.size(15).setColor(kGrey10),
+                    counterStyle: MyFonts.w500.size(12).setColor(kWhite),
+                    counterText:
+                        (title.text == "") ? "" : "${title.text.length}/20"),
+                validator: (value) {
+                  if (value == "" || value == null) {
+                    return "Field cannot be empty";
+                  }
+                  return null;
+                },
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(5.0),
+              child: Text(
+                'Body',
+                style: MyFonts.w500.setColor(kWhite),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(5.0),
+              child: TextFormField(
+                style: MyFonts.w500.size(15).setColor(kWhite),
+                controller: body,
+                maxLength: 150,
+                maxLines: 4,
+                decoration: InputDecoration(
+                    border: OutlineInputBorder(
+                        borderRadius: BorderRadius.circular(8)),
+                    fillColor: kAppBarGrey,
+                    filled: true,
+                    hintStyle: MyFonts.w500.size(15).setColor(kGrey10),
+                    counterStyle: MyFonts.w500.size(12).setColor(kWhite),
+                    counterText:
+                        (title.text == "") ? "" : "${title.text.length}/150"),
+                validator: (value) {
+                  if (value == "" || value == null) {
+                    return "Field cannot be empty";
+                  }
+                  return null;
+                },
+              ),
+            ),
+            ElevatedButton(
+                onPressed: () {
+                  bool isValid = formKey.currentState!.validate();
+                  if (!isValid) {
+                    return;
+                  }
+                  Map<String,String> data = {'title': title.text, 'body':body.text,'type':selected};
+                  APIService.postFeedbackData(data);
+                },
+                child: const Text('Submit'))
```

Comment:
Center the submit button and make it let it look similar to the lost & found button.